telemetry/definitions/commonDefinitions.json

```
@@ -15,6 +15,11 @@
             "type": "double",
             "description": "Total time from start to end"
         },
+        {
+            "name": "amazonQFileIgnoredCount",
```

```suggestion
            "name": "count",
```

this will avoid needing to add a new field every time a metric wants to count something